app/main.py

```diff
@@ -16,4 +16,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+def create_sorted_variables(*args) -> dict:
```

Comment:
The function name 'create_sorted_variables' is not very descriptive. It's generally a good practice to name functions in a way that clearly describes what they do. Consider renaming it to something more descriptive, like 'sort_variables_by_mutability'.